### PR TITLE
Feature/patro23 50 add console application insights loggingg

### PR DESCRIPTION
**Pull request by KlaudiaGajocha**

### CODE REVIEW

File: src/api/app/Program.cs [hunk 1]
```diff
@@ -48,9 +49,32 @@
...
+builder.Logging.AddApplicationInsights(
+		configureTelemetryConfiguration: (config) =>
+			config.ConnectionString = builder.Configuration.GetConnectionString("APPLICATIONINSIGHTS_CONNECTION_STRING"),
+		configureApplicationInsightsLoggerOptions: (options) => { });
+
+builder.Logging.AddFilter<ApplicationInsightsLoggerProvider>("category", Microsoft.Extensions.Logging.LogLevel.Trace);
+
+using var channel = new InMemoryChannel();
+
+var app = builder.Build();
```
Comment (mateuszgiedzinski) [inline src/api/app/Program.cs:68]: This line should be where it was before.

File: src/api/app/Program.cs [hunk 2]
```diff
@@ -48,9 +49,32 @@
...
+
 builder.Services.AddScoped<ICommandBus, CommandBus>();
 builder.Services.AddScoped<IQueryBus, QueryBus>();
 
+app.MapGet("/Test", async context =>
```
Comment (mateuszgiedzinski) [inline src/api/app/Program.cs:73]: I think this /Test endpoint is not needed.

File: src/api/app/Program.cs [hunk 3]
```diff
@@ -48,9 +49,32 @@
 builder.Services.AddFromAssemblies(typeof(ICommandHandler<>));
 builder.Services.AddFromAssemblies(typeof(IQueryHandler<,>));
 
+using var loggerFactory = LoggerFactory.Create(builder =>
+{
+	builder.AddSimpleConsole(i => i.ColorBehavior = LoggerColorBehavior.Disabled);
+});
+
+var logger = loggerFactory.CreateLogger<Program>();
```
Comment (mateuszgiedzinski) [inline src/api/app/Program.cs:57]: I think logger is not necessary here .

Comment (mateuszgiedzinski) [reply src/api/app/Program.cs:57]: It is created only for Test endpoint and can be removed.

**Review by mateuszgiedzinski — CHANGES_REQUESTED** [review]

File: src/api/app/Program.cs [hunk 4]
```diff
@@ -4,20 +4,21 @@
...
 using Microsoft.AspNetCore.HttpLogging;
+using Microsoft.Extensions.Logging.Console;
+using Microsoft.Extensions.Logging.ApplicationInsights;
+using Microsoft.ApplicationInsights.Channel;
 using Microsoft.AspNetCore.Mvc.Authorization;
+using Intive.Patronage2023.Shared.Infrastructure.Commands.CommandBus;
 
 var builder = WebApplication.CreateBuilder(args);
-
-string corsPolicyName = "CorsPolicy";
```
Comment (mateuszmuszynski) [inline src/api/app/Program.cs:18]: Why has it been removed?

**Review by mateuszmuszynski — CHANGES_REQUESTED** [review]

File: src/api/app/Program.cs (hunk 4 above)
Comment (KlaudiaGajocha) [reply src/api/app/Program.cs:18]: The variable 'corsPolicyName' was assigned but its value was never used

File: src/api/app/Program.cs (hunk 3 above)
Comment (KlaudiaGajocha) [reply src/api/app/Program.cs:57]: it's removed

File: src/api/app/Program.cs (hunk 4 above)
Comment (mateuszgiedzinski) [reply src/api/app/Program.cs:18]: Because you removed this line builder.Services.AddCors(builder.Configuration, corsPolicyName);

**Review by mateuszgiedzinski — CHANGES_REQUESTED** [review]

File: src/api/app/Program.cs (hunk 4 above)
Comment (KlaudiaGajocha) [reply src/api/app/Program.cs:18]: Ok, I will correct it

**Review by LukaszWinowicz — APPROVED** [review]

**Review by mateuszmuszynski — APPROVED** [review]

**Review by mateuszgiedzinski — APPROVED** [review]

**Pull request merged**